app/utils/__tests__/DataUtils.spec.js

```diff
@@ -621,4 +624,30 @@ describe('Utils: DataUtils', () => {
       expect(getTranslatedProperty(item, 'name')).to.equal('Finnish name');
     });
   });
+
+  describe('getReservation functions', () => {
```

Comment:
- [ ] You could remove this describe block and instead have the `getCurrentReservation` and `getNextReservation` describe blocks on this level each with their own test setup.